services/graphql-server/src/graphql/definitions/platform/content/types/contact.js

```diff
@@ -63,7 +63,7 @@ input ContentContactOwnedContentInput {
 
 input ContentContactSortInput {
   field: ContentContactSortField = id
-  order: SortOrder = desc
+  order: SortOrder = values
```

Comment:
Could the order input value from the website be changed? Changing the default here will change it for everyone, not just the requesting customer.

Reply:
It could, that was part of my suggestion in the main PR comment: "Alternatively adjust here: https://github.com/parameter1/base-cms/blob/master/packages/marko-web-theme-monorail/graphql/fragments/content-page.js#L129 or utilize a tenant/site level defined fragment that changes the sort order as well."

Likely depends on to what scope is this considered "broken" or if we want to change it for everyone etc. @brandonbk thoughts?